corgidrp/astrom.py

```diff
@@ -676,33 +683,49 @@ def boresight_calibration(input_dataset, field_path='JWST_CALFIELD2020.csv', thr
         corgidrp.data.AstrometricCalibration: Astrometric Calibration data object containing image center coords in (RA,DEC), platescale, and north angle
         
     """
-    # load in the data
+    # load in the data considering multiple frames in the data
     dataset = input_dataset.copy()
-    image = dataset[0].data
-
-    # call the target coordinates from the image header
-    target_coordinate = (dataset[0].pri_hdr['RA'], dataset[0].pri_hdr['DEC'])
 
     # find sources in image and match to field
     if field_path == 'JWST_CALFIELD2020.csv':
-        dir_name = os.path.dirname(__file__)
-        full_field_path = os.path.join(dir_name, "tests/test_data", field_path)
+        full_field_path = os.path.join(os.path.dirname(os.path.dirname(__file__)), "tests/test_data", field_path)
```

Comment:
because this file is required for this function, and not just for tests, we need to move this function into corgidrp/corgidrp (same directory as this file). Can you make a folder called like "data" inside of corgidrp/corgidrp, and put JWST_CALFIELD2020.csv there instead? Also update MANIFEST.in so that this file is in the MANIFEST.